src/mk.sh

```diff
@@ -45,8 +45,8 @@ function mk_build
 
   PARALLEL_CORES=$(( $PARALLEL_CORES * 2 ))
```

Comment:
If you have time, can you fix this in another PR? See: https://github.com/rodrigosiqueira/kworkflow/issues/85